src/odemis/driver/test/actuator_test.py

```diff
@@ -1676,7 +1677,9 @@ def move_to_initial_position():
         time.sleep(0.1)
         cancelled = f.cancel()
         self.assertTrue(cancelled)
+
         # Assert that stage position is neither initial nor target positions
+        time.sleep(0.01)  # give some time to update all the positions
```

Comment:
To be clear, the simulator does end up at a position which is half-way. The only problem was that we were checking too soon, so the .position VA hadn't been updated yet (and so still showed the initial position). 
With this sleep the test passes. So I'd say it's fine?

Reply:
The simulator does end up at a position half-way. So both asserts will pass!
The only issue was that .position is immediately updated, so it would still contain the original value.

Reply:
Not sure why it reaches the target position when I try the test!

Reply:
By changing the sleep time, I was able to recreate the issue on my computer. Indeed, sometimes the simulator would think it wasn't cancelled and report it reached the target.
I've updated this PR to fix this bug too.